### PR TITLE
Include lines in transaction's refund requests

### PR DESCRIPTION
**Pull request by korycins**
I want to merge this change because it introduces a feature described in https://github.com/saleor/saleor/issues/12831

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are either absent or guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migrations are either absent or optimized for zero downtime
* [ ] The changes are covered by test cases

### CODE REVIEW

File: saleor/graphql/payment/mutations.py [hunk 1]
```diff
@@ -1468,6 +1458,101 @@ def perform_mutation(cls, root, info: ResolveInfo, /, **data):
         return TransactionRequestAction(transaction=transaction)
 
 
+class TransactionRequestRefundForGrantedRefund(BaseMutation):
```
Comment (fowczarek) [inline saleor/graphql/payment/mutations.py:1461]: Due to Codecov, there are missing tests for this mutation.

File: saleor/graphql/payment/tests/mutations/transaction_request_refund_for_granted_refund.py [hunk 2]
```diff
@@ -0,0 +1,378 @@
...
+import graphene
+from mock import patch
+
+from saleor.graphql.core.enums import TransactionRequestRefundForGrantedRefundErrorCode
+from saleor.graphql.core.utils import to_global_id_or_none
+from saleor.graphql.tests.utils import assert_no_permission, get_graphql_content
+from saleor.payment import TransactionAction, TransactionEventType
+from saleor.payment.interface import TransactionActionData
+from saleor.payment.models import TransactionEvent
+from saleor.webhook.event_types import WebhookEventSyncType
```
Comment (fowczarek) [inline saleor/graphql/payment/tests/mutations/transaction_request_refund_for_granted_refund.py:12]: Absolute imports. Saleor uses relative once.

File: saleor/graphql/payment/tests/mutations/transaction_request_refund_for_granted_refund.py [hunk 3]
```diff
@@ -0,0 +1,378 @@
+from decimal import Decimal
```
Comment (fowczarek) [inline saleor/graphql/payment/tests/mutations/transaction_request_refund_for_granted_refund.py:1]: `Codecov` mark this file as untested. Maybe it's related to why mutation `TransactionRequestRefundForGrantedRefund` is marked as untested.

**Review by fowczarek — APPROVED** [review]

File: saleor/graphql/payment/tests/mutations/transaction_request_refund_for_granted_refund.py (hunk 3 above)
Comment (korycins) [reply saleor/graphql/payment/tests/mutations/transaction_request_refund_for_granted_refund.py:1]: Yeah, the test file has incorrect name (missing `test_` preffix

**Review by 8r2y5 — APPROVED** [review]

**Review by IKarbowiak — APPROVED** [review]

**Pull request merged**